File: js/11d.js
//11d
const prompt = require("prompt-sync")({ sigint: true });
// define vars
let rating = "null";
let userAge = prompt();
// if statements
if (userAge >= 18){
    rating = "Anything";
} else if (userAge >= 15 && userAge < 18){
    rating = "up to a 15";
} else if (userAge >= 12 && userAge < 15){
    rating = "up to a 12";
} else {
    rating = "that your mam will be called to pick you up"
}
// print
console.log(`You can watch ${rating}`);

File: kotlin/11d.kt
fun main() {
    var rating : String
    val userAge : Int = readln().toIntOrNull() ?: 0;
    if (userAge >= 18){
        @Override
        rating = "anything";
    } else if (userAge >= 15){
        @Override
        rating = "15+";
    } else if (userAge >= 12){
        @Override
        rating = "12+";
    }else {
        @Override
        rating = "that your mam will pick you up";
    }
    println("You can see ${rating}");
}